packages/react/README.md

```diff
@@ -101,3 +119,593 @@ function WidgetDeleter() {
+  - `filter`: A list of filters to limit the set of returned records down to. Optional. See the Model Filtering section in your application's API documentation to see the available filters for your models.
+  - `search`: A search string to match backend records against. Optional. See the Model Searching section in your application's API documentation to see the available search syntax.
+  - `sort`: A sort order to return backend records by. Optional. See the Sorting section in your application's API documentation for more info.
+  - `first` & `after`: Pagination arguments to pass to fetch a subsequent page of records from the backend. `first` should hold a record count and `after` should hold a string cursor retrieved from the `pageInfo` of the previous page of results. See the Model Pagination section in your application's API documentation for more info.
+  - `last` & `before`: Pagination arguments to pass to fetch a subsequent page of records from the backend. `last` should hold a record count and `before` should hold a string cursor retrieved from the `pageInfo` of the previous page of results. See the Model Pagination section in your application's API documentation for more info.
+  - `requestPolicy`: The `urql` request policy to make the request with. See [`urql`'s docs](https://formidable.com/open-source/urql/docs/api/urql/);
+  - `pause`: Should the hook make a request right now or not. See [`urql`'s docs](https://formidable.com/open-source/urql/docs/api/urql/)
+
+`useFindMany` returns two values: a result object with the `data`, `fetching`, and `error` keys for use in your React component's output, and a `refetch` function to trigger a refresh of the hook's data.
+
+- `data: GadgetRecord[] | null`: The resulting page of records fetched from the backend for your model, once they've arrived
+- `fetching: boolean`: A boolean describing if the hook is currently making a request to the backend.
+- `error: Error | null`: An error from the client or server side, if encountered during the request. See the [Errors](#errors-from-the-returned-error-object) section.
+
+The `refetch` function returned as the second element can be executed in order to trigger a refetch of the most up to date data from the backend. See [`urql`'s docs](https://formidable.com/open-source/urql/docs/basics/react-preact/#reexecuting-queries) on re-executing queries for more information.
+
+Without any options, `useFindMany` will fetch the first page of backend records sorted by id. As the fetch happens, the hook will cause your component to rerender when the data or error arrives:
+
+```javascript
+import React from "react";
+import { api } from "../api"; // some file that instantiates @gadget-client/your-app-slug
+
+export const ShowWidgetNames = () => {
+  const [{ data, fetching, error }, _refetch] = useFindMany(api.widget);
+
+  if (fetching) return "Loading...";
+  if (error) return `Error loading data: ${error}`;
```

Comment:
Not sure if, for all the examples, you want to deal with `!data`, which is typical when `pause: true`.